incubator/exporter-prometheus/src/main/java/io/aklivity/zilla/runtime/exporter/prometheus/internal/config/PrometheusEndpointConfig.java

```diff
@@ -16,15 +16,15 @@
 
 import io.aklivity.zilla.runtime.engine.config.OptionsConfig;
 
-public class EndpointConfig extends OptionsConfig
+public class PrometheusEndpointConfig extends OptionsConfig
 {
-    public static final EndpointConfig DEFAULT = new EndpointConfig("http", 9090, "/metrics");
+    public static final PrometheusEndpointConfig DEFAULT = new PrometheusEndpointConfig("http", 9090, "/metrics");
```

Comment:
After giving this more thought, I think it's better not to hide such defaults in the code, would rather they are required to be explicit in the `zilla.yaml` file, especially when it comes to network communication and binding ports.

Perhaps it still makes sense to default the `path` to `/metrics` if that is something that can be omitted by default from `prometheus` setup when it is polling for metrics.

Reply:
done